scripts/skills/effects/rf_encumbrance_effect.nut

```diff
@@ -31,70 +31,29 @@ this.rf_encumbrance_effect <- ::inherit("scripts/skills/skill", {
 			text = "Current Encumbrance Level: [color=" + ::Const.UI.Color.NegativeValue + "]" + level + "[/color]"
 		});
 
-		switch (level)
+		local fatigueBuildUp = -this.getFatigueRecoveryModifier(level);
```

Comment:
I'd call this `getFatigueOnTurnStart(level)`. That way it's the function's job to return the correct value and sign and we don't have to do this inversion. It also makes it clear that it's not a modifier to the `FatigueRecovery` property but rather is actually modifying Fatigue at turn start.